codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -185,6 +186,20 @@ public final T build(Http2Connection connection) {
                 reader = new Http2InboundFrameLogger(reader, frameLogger);
                 writer = new Http2OutboundFrameLogger(writer, frameLogger);
             }
+
+            // Configure the local flow controller.
+            if (connection.local().flowController() == null) {
```

Comment:
Should we just revert this change now that the builder not longer includes any new options? Just leave the responsibility of setting the flow controller in the decoder/encoder if not set?